src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1862,11 +1862,11 @@
                       Grid.Row="2"
                       Grid.RowSpan="2"
                       Height="Auto"
-                      Width="0"
+                      Width="3"
                       Name="extensionSplitter"
                       HorizontalAlignment="Stretch"
                       VerticalAlignment="Stretch"
-                      Margin="0"
+                      Margin="-2"
```

Comment:
What do you think about this?

Reply:
Well this will mean user will only have 1 px to drag? I am leaning towards making it consistent with the other splitter, you can tell from line 1855

Reply:
I think the user will still have 3px of splitter but the splitter element will be moved to the left and would overlap a little with the workspace but the color is transparent so no difference in the view.

Reply:
a cool solution but googling a bit on this bring up some exceptions in wpf-  I wonder if it's best to avoid negative margins.

Reply:
Yes, I will remove the negative margin.